app/templates/resources/view.html

```diff
@@ -0,0 +1,20 @@
+{% extends 'layouts/base.html' %}
+
+{% block content %}
+    <div class="ui stackable centered grid container">
+        <div class="twelve wide column">
+            <table class="ui compact definition table">
+                <tr><td>Name</td><td>{{ resource.name }}</td></tr>
+                <tr><td>Description</td><td>{{ resource.description }}</td></tr>
+                <tr><td>Website</td><td>{{ resource.website }}</td></tr>
+                <tr><td>Address</td><td>{{ address.street_address }}</td></tr>
+                <tr><td>User</td><td>{{ user.first_name }} {{ user.last_name }}</td></tr>
+                <tr><td>Reviews</td><td>
+                    {% for review in resource.reviews.all() %}
+                        {{ review }}
```

Comment:
does this look like the picture you posted in the pull request description? I feel like it should actually print the `__repr__` method of the review, not a sql query?